packages/data-context/src/actions/WizardActions.ts

```diff
@@ -190,13 +190,15 @@ export class WizardActions {
 
     assert(chosenFramework && chosenLanguage && chosenBundler)
 
-    return await Promise.all([
+    const scaffoldedFiles = await Promise.all([
```

Comment:
Rather than maintain a list of the expected order of these files, can we just order them here and write a comment? We loop through this array to render out the files changed so if it was written like:
```
return await Promise.all([
      this.scaffoldConfig('component'),
      this.scaffoldSupport('component', chosenLanguage),
      this.scaffoldSupport('commands', chosenLanguage),
      this.scaffoldComponentIndexHtml(chosenFramework),
      this.scaffoldFixtures(),
    ])
```

the order would be correct and it would remove the need to sort them.

Reply:
💡 _work smarter, not harder_ lolllll this is so much simpler, thank you for pointing that out! Makin' those changes now!!

Reply:
Not sure we need `async/await` here, either.

```ts
private scaffoldComponent () {
  return Promise.all([ /* ... */ ])
```

Reply:
@lmiller1990 true that, updating!

Reply:
Actually I'mma keep it because all things being equal, it was helpful to have an intermediate variable for `scaffoldedFiles` that could be easily console-logged after the `await`